driver/tke_driver.go

```diff
@@ -480,26 +471,38 @@ func (d *Driver) getWrapCreateClusterRequest(state *state) (*ccs.CreateClusterRe
 	return request, nil
 }
 
-func (d *Driver) waitTKECluster(ctx context.Context, svc *ccs.Client, state *state) error {
+func waitTKECluster(ctx context.Context, svc *ccs.Client, state *state) error {
 	lastMsg := ""
+	timeout := time.Duration(30 * time.Minute)
+	tick := TickerContext(ctx, 15*time.Second)
```

Comment:
You need to use the **TimeOut** context for your ticker instead of the parent context. The ticker should stop when the TimeOut context is done.

Reply:
updated, thanks.